scripts/docker-crontab

```diff
@@ -9,4 +9,8 @@ SHELL="/bin/bash"
 USER="geodata"
 
 */5 * * * * bash /index-sync-rotate.sh
+
+# truncate sphinx logs
+0 0 * * * : > /var/log/sphinxsearch/searchd.log
```

Comment:
With this we'd completely empty the logs every night and all information is lost.
Would it be an option to really use `truncate` and only delete the oldest parts of the logs, so that the files are reduced to a defined size every night?

Could the information in those files be of use in case we detect some problems a few days after when they happened and use the old logs for debugging?

Reply:
everything is copied via stdout of the main process to the logfile of the docker container and via filebeat to kibana.
the tail -f in the docker-cmd print the content of these truncated logs in realtime to the main process stdout.

the log content is already copied to docker / filebeat / kibana before the file will be truncated.

Reply:
the advantage of this approach is that we dont have to run the container as root and now the main process searchd has pid 1 which is also better i think.

Reply:
with the fifo that have been created for cron it is now possible to run any command inside the container and redirect the output to docker logs
```bash
df -h > /tmp/stdout
```

Reply:
OK, makes sense, the information is in kibana of course.
So it makes completely sense to delete it locally every night :+1: